src/abstract_model.cpp

```diff
@@ -654,4 +654,32 @@ void AbstractModel::fdwdw_rowvals(SUNMatrixWrapper &/*dwdw*/) {
                        __func__);
 }
 
+void AbstractModel::fdx_rdatadx_solver(realtype */*dx_rdatadx_solver*/,
+                               const realtype */*x*/, const realtype */*tcl*/,
+                               const realtype */*p*/, const realtype */*k*/)
+{
+    throw AmiException("Requested functionality is not supported as %s is "
```

Comment:
only throw if nx_solver != nx_rdata ?

Reply:
At the moment this is checked in `fsx_rdata`. If `nx_solver == nx_rdata`, this will never be called. I am not sure where else this might be required, so I thought it might be better to just throw.